sdks/java/extensions/euphoria/euphoria-core/src/test/java/org/apache/beam/sdk/extensions/euphoria/core/translate/JoinTest.java

```diff
@@ -1,358 +0,0 @@
-/*
```

Comment:
I would rather integrate those tests into testkit than delete them (the same for `FlatMapTest`, `ReduceByKeyTest`).

Reply:
I think they test the same things that are already tested in testkit (they were used for early stage of beam translation prototyping). We should upgrade testkit to support more translators (eg join suite should run against multiple translators) in other issue.